static/src/stylesheets/module/experiments/_embed.scss

```diff
@@ -80,6 +80,7 @@
 
 .contributions__payment-logos {
     height: 25px;
+    svg, img { height: 100%; width: auto; }
```

Comment:
the SVG has a specific width and height, so this overrides it back to scale proportionally with the container